src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -46,8 +46,12 @@ const ScopeSelectionScreen = ({ navigation, route }: IntroScreenProps): JSX.Elem
 
   useFocusEffect(refreshSelectedProfessions)
 
+  useLayoutEffect(() => {
+    navigation.setOptions({ headerShown: !initialSelection })
+  })
```

Comment:
Shouldn't we use callbacks for `useLayoutEffect`? Or does this just not matter since nothing happens if the options are already set?

Reply:
this only gets triggered once in any case, no? so i think its not needed to useCallback. To be honest i didn't use this hook so far ;)

Reply:
React Navigation Docs do it without callback, so I think it is fine to use it without.